include/itkBlockMatchingStrainWindowBlockAffineTransformCommand.hxx

```diff
@@ -70,13 +70,15 @@ StrainWindowBlockAffineTransformCommand<TStrainWindowDisplacemenCalculator,
 
   if (m_UseStrainWindowStrain)
   {
-    m_BlockAffineTransformMetricImageFilter->SetStrainImage(strainWindower->GetStrainImageFilter()->GetOutput());
+    m_BlockAffineTransformMetricImageFilter->SetStrainImage(
+          const_cast<typename BlockAffineTransformMetricImageFilterType::StrainImageType *>(strainWindower->GetStrainImageFilter()->GetOutput()));
```

Comment:
Is `m_BlockAffineTransformMetricImageFilter` operating on the strain image passed in? If not, would it be better to adjust that filter so that `SetStrainImage` takes a `const` image rather than using `const_cast` to cast away output constness?

Reply:
Due to the limitations of const-correctness from the beginning of ITK/VTK days in how filters handle objects, we need the `const_cast` here.

Reply:
Thanks for the review :pray: 

@thewtex @tbirdso  I was able to update the `SetStrainImage` setter to accept a const image ... See [cb285e0](https://github.com/KitwareMedical/ITKUltrasound/pull/198/commits/cb285e01859f8d7e818aafd178fae328bffec731)

Reply:
Excellent, thank you @jcfr ! 🚀